code/game/machinery/doors/airlock.dm

```diff
@@ -108,6 +109,7 @@
 	opacity = 1
 	assembly_type = /obj/structure/door_assembly/door_assembly_highsecurity //Until somebody makes better sprites.
 	explosion_block = 2
+	is_special = 1

```

Comment:
The vault shouldn't be immune to this; it's hackable and openable by the AI alike--there's nothing particularly special about it.